go.mod

```diff
@@ -9,8 +9,8 @@ require (
 	github.com/Microsoft/hcsshim v0.9.2 // indirect
 	github.com/NYTimes/gziphandler v0.0.0-20170623195520-56545f4a5d46
 	github.com/aquasecurity/bolt-fixtures v0.0.0-20200903104109-d34e7f983986
-	github.com/aquasecurity/fanal v0.0.0-20220225095822-ef150f781751
-	github.com/aquasecurity/go-dep-parser v0.0.0-20220224134419-e4f58c60089e
+	github.com/aquasecurity/fanal v0.0.0-20220302151637-0570589bca94
+	github.com/aquasecurity/go-dep-parser v0.0.0-20220302151315-ff6d77c26988
```

Comment:
I have a question here:
why both `trivy` and `fanal` have dependencies on `go-dep-parser`

Reply:
We set logger for go-dep-parse in Trivy.
https://github.com/aquasecurity/trivy/blob/1b66b77f691f2249a63c284056afcad704c0e15f/pkg/log/logger.go#L11